tests/__fixtures__/checkout/checkout.fixtures.mts

```diff
@@ -1054,6 +1054,8 @@ export const deliveryBundlesEntity = {
   },
 };
 
+export const mockDeliveryBundlesEntityDenormalized = [deliveryBundle];
```

Comment:
This still doesn't look okay. Aren't we expecting two delivery bundles in the response?

Reply:
I agree with @anapim. This seems to be not ok. 
Did you tried to add
```js
export const mockDeliveryBundlesEntityDenormalized = [
   deliveryBundle,
   {
    ...deliveryBundle,
    id: '090998',
    name: 'fake bundle',
    isSelected: false,
  }
];
```

I think this should be the result....

Reply:
I found the issue. On line 1000, we're missing the second delivery bundle id.

```javascript
export const checkoutEntity = {
  checkoutOrder: checkoutOrderId,
  id: checkoutId,
  shippingOptions: [shippingOption],
  deliveryBundles: [deliveryBundleId, '090998'],
  orderStatus: OrderStatusError.NoError,
};
```

Then @ivomota suggestions should work.

Reply:
I made the adjustment. I also found the error in checkoutEntity (line 1000). I needed to add another delivery package id in checkoutEntity. However, when I run the test, I see an error in the redux checkout selectors. I have also solved this problem.